### PR TITLE
fix!: Store some JSON columns as jsonb not as a text

### LINKED ISSUE #2430: AWS Data Types
### Describe the Bug

Storing policy data as a text rather than JSONB

### Expected Behavior

Data should be stored as a JSONB object or an array rather than text
According to the [migration document](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/aws/docs/v1-migration.md) the following fields have changed in ways that we need to fix (or at least validate)
- [x] `aws_codepipeline_pipelines.tags` --> Type changed from jsonb to text
- [x] `aws_dax_clusters.tags` --> Type changed from jsonb to text
- [x] `aws_ecr_repositories.tags` --> Type changed from jsonb to text
- [x] `aws_route53_traffic_policy_versions.document` --> Type changed from jsonb to text
- [x] `aws_sns_subscriptions.delivery_policy` --> Type changed from jsonb to text
- [x] `aws_sns_subscriptions.effective_delivery_policy` --> Type changed from jsonb to text
- [x] `aws_sns_subscriptions.filter_policy` --> Type changed from jsonb to text
- [x] `aws_sns_subscriptions.redrive_policy` --> Type changed from jsonb to text
- [x] `aws_sns_topics.delivery_policy` --> Type changed from jsonb to text
- [x] `aws_sns_topics.effective_delivery_policy` --> Type changed from jsonb to text
- [x] `aws_sns_topics.policy` --> Type changed from jsonb to text
- [x] `aws_sqs_queues.policy` --> Type changed from jsonb to text
- [x] `aws_sqs_queues.redrive_allow_policy` --> Type changed from jsonb to text
- [x] `aws_sqs_queues.redrive_policy` --> Type changed from jsonb to text

### PR DESCRIPTION
**Pull request by amanenk**
#### Summary

closes #2430

### CODE REVIEW

File: plugins/source/aws/resources/services/rds/db_security_groups.go [hunk 1]
```diff
@@ -47,7 +47,7 @@ func DbSecurityGroups() *schema.Table {
 				Resolver: schema.PathResolver("DBSecurityGroupName"),
 			},
 			{
-				Name:     "ec2_security_groups",
+				Name:     "e_c2_security_groups",
```
[the remark this answers is not in the document]
Comment (hermanschaaf) [reply plugins/source/aws/resources/services/rds/db_security_groups.go:50]: https://github.com/cloudquery/cloudquery/pull/2473

**Review by hermanschaaf — CHANGES_REQUESTED** [review]
LGTM, we just need to rebase and regenerate using latest main branch so the `ec2` column name doesn't change

File: plugins/source/aws/codegen/recipes/sqs.go [hunk 2]
```diff
@@ -27,6 +26,21 @@ func SQSResources() []*Resource {
...
+						Type:     schema.TypeJSON,
+						Resolver: `client.MarshaledJsonResolver("Policy")`,
+					},
+					{
+						Name:     "redrive_policy",
+						Type:     schema.TypeJSON,
+						Resolver: `client.MarshaledJsonResolver("RedrivePolicy")`,
+					},
+					{
+						Name:     "redrive_allow_policy ",
```
[the remark this answers is not in the document]
Comment (amanenk) [reply plugins/source/aws/codegen/recipes/sqs.go]: we might want to add a check for it

[the remark this answers is not in the document]
Comment (hermanschaaf) [reply plugins/source/aws/codegen/recipes/sqs.go]: Yeah, we should validate it with a regex

**Review by hermanschaaf — APPROVED** [review]

**Pull request merged**